_drafts/SC1-2-2-video-captions.md

```diff
@@ -0,0 +1,88 @@
+---
+name: video captions
+description: |
+  Captions are available for audio information in non-streaming `video` elements.
+
+success_criterion:
+- 1.2.2 # Captions (Prerecorded)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+- [Audio output][]
+- [Visual output][]
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming][] `video` element [visible on the page][] where the video contains [audio][].
+
+### Expectation
+
+Of each test target, audio information that is not expressed visually in the video, is available through [captions][].
+
+*Note*: Captions can be either embedded in the video file itself or can be made available trough a separate track.
```

Comment:
I think it would make sense to split this out into separate atomic rules in a composed rule, with one atomic rule for each of the methods for creating captions, to allow for automating as much as possible.

Reply:
... and then maybe split the one expectation out into two, one checking that there are captions at all, and another one checking that the captions are appropriate. 
(so that you can get points for getting the concept of captions (e.g. using youtube's auto-captions), and points for actually getting the content right)

Reply:
Don't know if the two expectations should actually be two separate rules... (one might be semi-automatable, the other one not at all)

Reply:
We started off with separate rules but found that the two were far too similar to warrant two rules. The only difference between them was two or three words. At that point, I don't think separate rules make sense. I'm not sure that there's much to be automated about this rule anyway.